tests/bgp/test_bgp_queue.py

```diff
@@ -10,36 +10,37 @@
 ]
 
 
-def clear_queue_counters(duthost):
-    duthost.shell("sonic-clear queuecounters")
+def clear_queue_counters(asichost):
+    asichost.command("sonic-clear queuecounters")
 
 
-def get_queue_counters(duthost, port, queue):
+def get_queue_counters(asichost, port, queue):
     """
     Return the counter for a given queue in given port
     """
     cmd = "show queue counters {}".format(port)
-    output = duthost.shell(cmd)['stdout_lines']
+    output = asichost.command(cmd)['stdout_lines']
     txq = "UC{}".format(queue)
     for line in output:
         fields = line.split()
         if fields[1] == txq:
-            return int(fields[2])
+            return int(fields[2].replace(',', ''))
     return -1
 
 
 def test_bgp_queues(duthosts, enum_frontend_dut_hostname, enum_asic_index, tbinfo):
     duthost = duthosts[enum_frontend_dut_hostname]
-    clear_queue_counters(duthost)
+    asichost = duthost.asic_instance(enum_asic_index)
+    clear_queue_counters(asichost)
     time.sleep(10)
     bgp_facts = duthost.bgp_facts(instance_id=enum_asic_index)['ansible_facts']
     mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
```

Comment:
do we need to the minigraph facts for asichost as well?

Reply:
> do we need to the minigraph facts for asichost as well?

Hi @arlakshm, fixed.